plugin.video.gamekings/resources/lib/gamekings_main.py

```diff
@@ -27,6 +27,18 @@ def __init__(self):
         # Get the plugin handle as an integer number
         self.plugin_handle = int(sys.argv[1])
 
+        #
+        # Premium Videos
+        #
+        parameters = {"action": "list", "plugin_category": LANGUAGE(30005),
+                      "url": BASE_URL_GAMEKINGS_TV + "category/premium/page/001/", "next_page_possible": "True"}
+        url = self.plugin_url + '?' + urllib.parse.urlencode(parameters)
+        list_item = xbmcgui.ListItem(LANGUAGE(30005), iconImage="DefaultFolder.png")
```

Comment:
Remove iconimage kwarg

Reply:
Add it to setArt as "thumb" or "icon"

Reply:
Fixed.
Also fixed some other stuff:
- fix for trailers
- removed http:// urls